src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "advancement.mythicmetals.starrite.title": "Gefallene Sternentropfen",
+  "advancement.mythicmetals.steel.description": "Schmiede Stahl, eine einfache Legierung aus Eisen and Manganese",
+  "advancement.mythicmetals.steel.title": "Aber Stahl ist schwerer als...",
+  "advancement.mythicmetals.stormyx.description": "Finde Stormyx tief im Nether versteckt",
+  "advancement.mythicmetals.stormyx.title": "Magischee Metalle",
+  "advancement.mythicmetals.stormyx_shell.description": "Bekomme eine Stormyx Shell, selten aus Stormyxerzen",
+  "advancement.mythicmetals.stormyx_shell.title": "Der Nether Diamand",
+  "advancement.mythicmetals.stormyx_shield.description": "Crafte einen Stormyxschild aus einer Stormyx Shell",
+  "advancement.mythicmetals.stormyx_shield.title": "Gebet für besseren Schutz",
+  "advancement.mythicmetals.unobtainium_template.description": "Finde eine Unerhaltbarium-Schmiedenvorlage in einer Antiken Stadt, die es Ihnen ermöglicht, mächtige Ausrüstung aus Unerhaltbarium-Legierungen zu schmieden",
+  "advancement.mythicmetals.unobtainium_template.title": "Das Geheimnis des Sculk",
+  "advancement.mythicmetals.unobtainium.description": "Bekomme Unerhaltbarium, für gewöhnlich Tief im Untergrund zu finden",
+  "advancement.mythicmetals.unobtainium.title": "Wie hast du das bekommen?",
+  "advancement.mythicmetals.superhot.title": "Ich bin on Fire!",
+  "advancement.mythicmetals.superhot.description": "Versuche dich nicht in Holzkohle zu verwandeln, wenn du Verbrennung auf dich anwendest, was die Feuerresistenz umgeht und den erlittenen Feuerschaden erhöht.!",
+  "advancement.mythicmetals.intentional_combusting_mechanics.title": "Das ist Based",
+  "advancement.mythicmetals.intentional_combusting_mechanics.description": "Nutze die Mechanik des verweilenden Creepers, um die Verbrennung auf dich selbst anzuwenden.",
+
+  "abilities.mythicmetals.aqua_affinity": "Natürliche Wasseraffinität",
+  "abilities.mythicmetals.blast_mining": "Blast Mining",
+  "abilities.mythicmetals.blast_padding": "Blast Padding",
+  "abilities.mythicmetals.blast_protection": "Natürliche Blast Resistenz",
+  "abilities.mythicmetals.bonus_fortune": "Bonus Glück",
+  "abilities.mythicmetals.bonus_looting": "Bonus Plünderung",
+  "abilities.mythicmetals.carmot_shield": "+5 Carmotschild",
+  "abilities.mythicmetals.depth_strider": "Natürlicher Depth Strider",
+  "abilities.mythicmetals.hot": "Branding",
+  "abilities.mythicmetals.feather_falling": "Natürlicher Federfall",
+  "abilities.mythicmetals.firework_boost": "Feuerwerksboost",
+  "abilities.mythicmetals.fire_aspect": "Natürlicher Feuer Aspekt",
+  "abilities.mythicmetals.fire_protection": "Natürlicher Feuerschutz",
+  "abilities.mythicmetals.knockback": "Natürlicher Rückstoß",
+  "abilities.mythicmetals.material_tooltip": "Seltenes Crafting Material",
+  "abilities.mythicmetals.mending": "Regrowth",
```

Comment:
Untranslated. Regrowth is an auto-repair mechanic which slowly regenerates durability of gear over time